.github/ISSUE_TEMPLATE/feature_request.yml

```
@@ -0,0 +1,23 @@
+name: Feature Request
+description: Suggest an idea for Xournal+.
+labels: enhancement
+body:
+  - type: textarea
+    attributes:
+      label: Is Your Feature Request Related to a Problem?
+      description: A clear and concise description of what your problem is.
+      placeholder: Ex. I'm always frustrated when...
+  - type: textarea
+    attributes:
+      label: Feature Description
+      description: A clear and concise description of the feature that you would propose.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Alternatives That You've Considered
+      description: Add any alternative solutions that you may have considered here.
+  - type: textarea
+    attributes:
+      label: Additional Context
+      description: Put any extra contextextual information here.
```

```suggestion
      description: Add any other context about the problem here.
```

---

How about somewhat of a combination of the two:
```suggestion
      description: Add any other contextual information for the problem here.
```

---

Ok